.travis.yml

```diff
@@ -3,20 +3,31 @@ language: ruby
 sudo: false
 before_install:
   - gem install bundler
-  - "rm ${BUNDLE_GEMFILE}.lock"
+  - "rm -f ${BUNDLE_GEMFILE}.lock"
```

Comment:
Currently, lock file of each Gemfile does not add to repository. 
Therefore, in order to become an error in rm with `-f` option is not, it was added in order to avoid it.

Ref: https://travis-ci.org/y-yagi/globalid/jobs/144035006